Automattic/woocommerce-payments
# Update Github workflows to use node version in .nvmrc

#### Changes proposed in this Pull Request

The workflows `i18n-weekly-release` & `pre-release-publish-on-tag` doesn't use the node version specified in `.nvmrc` resulting in build errors. e.g. https://github.com/Automattic/woocommerce-payments/runs/4423843973?check_suite_focus=true

This PR fixes both the workflows to use the node version specified in `.nvmrc`.

#### Testing instructions

Since the changes are related to Github Actions workflow, there are no specific testing instructions. Confirm that the workflows run as expected without throwing errors.

Related PR - https://github.com/Automattic/woocommerce-payments/pull/3464

-------------------


**Post merge**
- [ ] Confirm the workflows run as expected - @achyuthajoy
